src/nn_skeleton.py

```diff
@@ -110,8 +100,73 @@ def __init__(self, mc):
     self.flop_counter = [] # array of tuple of layer name, flop number
     # activation counter
     self.activation_counter = [] # array of tuple of layer name, output activations
-    self.activation_counter.append(('input', mc.IMAGE_WIDTH*mc.IMAGE_HEIGHT*3))
+    self.activation_counter.append(('input', IMAGE_WIDTH*IMAGE_HEIGHT*3))
+
+  def _set_anchors_and_placeholders(self,preds):
```

Comment:
If you refer to line 103, the It is not a path. It is a list counting number of elements in a layer.